talk/objectorientation/objectsclasses.tex

```diff
@@ -13,7 +13,7 @@
   \begin{block}{Objects}
     instances of classes
   \end{block}
-  \begin{block}{A class encapsulates a concept}
+  \begin{block}{A class encapsulates state and behavior of ``something''}
```

Comment:
I just disliked "concept" here. Feel free to suggest something better!

Reply:
Indeed, now that concept has some other meaning in C++, it does not feel right anymore. I'm happy with "something" !